blockdag/validate.go

```diff
@@ -471,105 +471,164 @@ func (dag *BlockDAG) checkBlockSanity(block *util.Block, flags BehaviorFlags) (t
 	if err != nil {
 		return 0, err
 	}
+	err = dag.checkBlockContainsAtLeastOneTransaction(msgBlock)
```

Comment:
Please leave as is

Reply:
We've spoken about this and decided to do it the current way.